src/abstract-sql-utils.ts

```diff
@@ -1,9 +1,9 @@
-import { Definition } from '@resin/odata-to-abstract-sql';
+import { readFileSync } from 'fs';
+import * as _ from 'lodash';
 
 import { AbstractSqlModel } from '@resin/abstract-sql-compiler';
+import { Definition } from '@resin/odata-to-abstract-sql';
 import { sbvrUtils } from '@resin/pinejs';
-import { readFileSync } from 'fs';
-import * as _ from 'lodash';
```

Comment:
You've reorganized this "by type" but you've grouped type-only imports with "real" imports which doesn't make sense imo (and those type-only imports should be `import type` but that wasn't possible until very recently)

Reply:
Yeah, I meant by "origin" (I mention "origin" everywhere except the commit title). I don't think separating type-only imports makes sense in this light.

Reply:
Marking type-only imports on #314